articles/finance/localizations/emea-deu-gdpdu-audit-data-export.md

```diff
@@ -123,7 +123,7 @@ The following tables show the Tax data structure definitions.
 |-----|------------------------|-----------|---------------------------------------------|--------------------------------------|
 | 1   | STEUERART              | Zeichen   | Beschreibung der Steuerart                  | $TaxTrans/taxName()                                      |
 | 2   | STEUERBUCHUNGSREFERENZ | Numerisch | Gibt es hierzu eine MWST-Buchung? - lfd Nr. | $TaxTrans/$TaxTransGeneralJournalAccountEntry/$GeneralJournalAccountEntryRecId                                          |
-| 3   | MWST\_CODE             | Zeichen   | MWST Bezeichung                             | $TaxTrans/TaxCode                                        |
+| 3   | MWST\_CODE             | Zeichen   | MWST Bezeichnung                            | $TaxTrans/TaxCode                                        |
```

Comment:
This is a German correction...
https://www.google.com/search?q=%22MWST+Bezeichung%22

> Showing results for ["MWST Bezeichnung"](https://www.google.com/search?sxsrf=APq-WBvrEzAh4orDyfaiyCr13LkxIpvxrw:1649991690685&q=%22MWST+Bezeichnung%22&spell=1&sa=X&ved=2ahUKEwiP6OuEipX3AhX4hHIEHcRVBxUQkeECKAB6BAgBEDI)
> Search instead for ["MWST Bezeichung"](https://www.google.com/search?sxsrf=APq-WBvrEzAh4orDyfaiyCr13LkxIpvxrw:1649991690685&q=%22MWST+Bezeichung%22&nfpr=1&sa=X&ved=2ahUKEwiP6OuEipX3AhX4hHIEHcRVBxUQvgUoAXoECAEQMw)